mongo_import_export_schema/mongo-schema-import.py

```diff
@@ -60,12 +82,14 @@ def main(argv=sys.argv):
     parser.add_argument('--password', metavar='pwd', type=str, help='Username', default='')
     parser.add_argument('--authSource', metavar='a', type=str, help='DB to auth against', default='admin')
     parser.add_argument('--file', metavar='f', type=str, help='Path to exported .json file', default='config.json')
-    parser.add_argument('--delete-db', action='store_true', help='Delete existing database if it exist', )
+    parser.add_argument('--delete-db', action='store_true', help='Delete existing database if it exist')
```

Comment:
Rephrase to "Delete database if it exists". Also, you add a new argument and never use it.

Reply:
this argument i didn't add it :) i just delete the coma ^^